### ISSUE TITLE
[BUG] Lich King's Remorseless Winter graphic is missing

### ISSUE THREAD

**Issue by raethkcj** [JamminL/wotlk-classic-bugs#792]
#### 1.1. Description
  In the Lich King encounter, when Remorseless Winter is cast, there is one animation on the cast, and then the graphic disappears.
  #### 1.2. How to Reproduce
  1. Fight the Lich King until 70% HP when Remorseless Winter is cast.
  #### 1.3  Source Material
https://youtu.be/JIiXfwH79iY?si=qFd6LIcLQ4uHk6-Q&t=4620
#### 2.1. Description
 There should be a persistent blue glow with pulsing white snow flurries while Remorseless Winter is active.
  #### 2.2 Source Material
https://youtu.be/_QcuB8zCpu4?si=ZEDva32gJ8wtbJaf&t=253

**Comment by Kolenich** [comment]
Will there be an update to this? I wouldn't label it as "critical", but bot "minir" either. It's a Lich King boss fight after all, a trademark of whole WotLK.

**Comment by raethkcj** [comment]
I tend to agree that this is not a minor bug. For a player who has not done the encounter nor watched a guide specifically on this mechanic, there's no indication whatsoever that standing in the area is a bad idea, let alone that it will do enough damage to 1-2 shot them.
